src/scraper/__main__.py

```diff
@@ -182,12 +245,133 @@ def scrape(self):
             if option.text
         }
         # if juidicial_officers param is not specified, use all of them
-        if not args.judicial_officers:
-            args.judicial_officers = list(judicial_officer_to_ID.keys())
+        if not judicial_officers:
+            judicial_officers = list(judicial_officer_to_ID.keys())
+        return judicial_officers, judicial_officer_to_ID
 
-        # initialize variables to time script and build a list of already scraped cases
-        START_TIME = time()
-        cached_case_list = [file_name.split(".")[0] for file_name in os.listdir(self.case_html_path)]
+    def scrape_results_page(self, odyssey_version, base_url, search_url, hidden_values, JO_id, date_string):
+        # POST a request for search results
+        results_page_html = request_page_with_retry(
+            session=self.session,
+            url=search_url
+            if odyssey_version < 2017
+            else urllib.parse.urljoin(base_url, "Hearing/SearchHearings/HearingSearch"),
+            verification_text="Record Count"
+            if odyssey_version < 2017
+            else "Search Results",
```

Comment:
This block trips me up.. I think this logic could be more clear?

Reply:
I am making a comment to myself to come back to this and change it, hah.